### PR TITLE
Mention availability of a built CSS file in the package

### PR DESCRIPTION
**Pull request by colinrotherham**
Closes https://github.com/alphagov/govuk-frontend-docs/issues/348

### CODE REVIEW

File: source/importing-css-assets-and-javascript/index.html.md.erb [hunk 1]
```diff
@@ -7,15 +7,31 @@ weight: 70
 
 ## CSS
 
-### Import all the CSS
+### Add the CSS file to your HTML
+
+If you decide to add the CSS to your HTML, first either:
```
Comment (claireashworth) [inline source/importing-css-assets-and-javascript/index.html.md.erb:12]: are the bullet points an either/or option or steps you have to do one after the other?

File: source/importing-css-assets-and-javascript/index.html.md.erb [hunk 2]
```diff
@@ -7,15 +7,31 @@ weight: 70
...
+
+- set up your routing so that requests for the CSS file are served from `node_modules/govuk-frontend/dist/govuk/govuk-frontend.min.css`
```
Comment (claireashworth) [inline source/importing-css-assets-and-javascript/index.html.md.erb:14]:
```suggestion
- set up your routing so requests for the CSS file are served from `node_modules/govuk-frontend/dist/govuk/govuk-frontend.min.css`
```

File: source/importing-css-assets-and-javascript/index.html.md.erb [hunk 3]
```diff
@@ -152,14 +168,14 @@ To import the JavaScript from GOV.UK Frontend, you can either:
 
 If you decide to add the JavaScript to your HTML, first either:
 
-- set up your routing so that requests for the JavaScript file are served from  `node_modules/govuk-frontend/dist/govuk/govuk-frontend.min.js`
+- set up your routing so that requests for the JavaScript file are served from `node_modules/govuk-frontend/dist/govuk/govuk-frontend.min.js`
```
Comment (claireashworth) [inline source/importing-css-assets-and-javascript/index.html.md.erb:171]:
```suggestion
- set up your routing so requests for the JavaScript file are served from `node_modules/govuk-frontend/dist/govuk/govuk-frontend.min.js`
```

File: source/install-using-precompiled-files/index.html.md.erb [hunk 4]
```diff
@@ -26,8 +26,8 @@ You’ll not be able to:
 1. Download the `release-<VERSION-NUMBER>.zip` file at the bottom of the [latest GOV.UK Frontend release note](https://github.com/alphagov/govuk-frontend/releases/latest).
 2. Unzip the zip file.
 3. Copy the `assets` folder to the root of your project’s public folder, so that for example `<YOUR-SITE-URL>/assets/govuk-crest.png` shows the `govuk-crest.png` image in your users’ browsers.
-4. Copy the 2 `.css` files to a stylesheets folder in the root of your project’s public folder, so that for example `<YOUR-SITE-URL>/stylesheets/govuk-frontend-<VERSION-NUMBER>.min.css` shows the CSS file in your users’ browsers.
-5. Copy the `.js` file to a JavaScript folder in the root of your project’s public folder, so that for example `<YOUR-SITE-URL>/javascripts/govuk-frontend-<VERSION-NUMBER>.min.js` shows the JavaScript file in your users’ browsers.
+4. Copy the `.css` and `.css.map` files to a stylesheets folder in the root of your project’s public folder, so that for example `<YOUR-SITE-URL>/stylesheets/govuk-frontend-<VERSION-NUMBER>.min.css` shows the CSS file in your users’ browsers.
```
Comment (claireashworth) [inline source/install-using-precompiled-files/index.html.md.erb:29]:
```suggestion
4. Copy the `.css` and `.css.map` files to a stylesheets folder in the root of your project’s public folder. This is so, for example, `<YOUR-SITE-URL>/stylesheets/govuk-frontend-<VERSION-NUMBER>.min.css` shows the CSS file in your users’ browsers.
```

File: source/install-using-precompiled-files/index.html.md.erb [hunk 5]
```diff
@@ -26,8 +26,8 @@ You’ll not be able to:
...
+5. Copy the `.js` and `.js.map` files to a JavaScripts folder in the root of your project’s public folder, so that for example `<YOUR-SITE-URL>/javascripts/govuk-frontend-<VERSION-NUMBER>.min.js` shows the JavaScript file in your users’ browsers.
```
Comment (claireashworth) [inline source/install-using-precompiled-files/index.html.md.erb:30]:
```suggestion
5. Copy the `.js` and `.js.map` files to a JavaScripts folder in the root of your project’s public folder. This is so, for example, `<YOUR-SITE-URL>/javascripts/govuk-frontend-<VERSION-NUMBER>.min.js` shows the JavaScript file in your users’ browsers.
```

File: source/importing-css-assets-and-javascript/index.html.md.erb (hunk 1 above)
Comment (colinrotherham) [reply source/importing-css-assets-and-javascript/index.html.md.erb:12]: Can do either of them

I copied this pattern from the JavaScript section further down which is nearly identical

Happy to improve both?

>### Add the JavaScript file to your HTML
>
>If you decide to add the JavaScript to your HTML, first either:

Comment (claireashworth) [reply source/importing-css-assets-and-javascript/index.html.md.erb:12]: OK. How about - 
If you decide to add the CSS to your HTML, you can choose to either:
Another option - 
If you decide to add the CSS to your HTML, you can do one of the following:
I'm basically just removing the 'first' so it doesn't sound like the start of a list you have to do in a specific order.

**Comment by colinrotherham** [comment]
Thanks @claireashworth, I've made all the updates. Can you take another look please?

**Review by claireashworth — APPROVED** [review]

**Comment by colinrotherham** [comment]
Thanks @claireashworth

Will hold off for feedback from @romaricpascal

But I'm happy we're now treating CSS and JS equally with two routes:

1. Add our built files directly in your HTML
2. Import and build our files into your existing code

File: source/importing-css-assets-and-javascript/index.html.md.erb [hunk 6]
```diff
@@ -7,15 +7,36 @@ weight: 70
...
+- import the CSS into your own Sass file
+
+### Add the CSS file to your HTML
+
+If you decide to add the CSS to your HTML, you can do one of the following:
+
+- set up your routing so requests for the CSS file are served from `node_modules/govuk-frontend/dist/govuk/govuk-frontend.min.css`
+- copy the `node_modules/govuk-frontend/dist/govuk/govuk-frontend.min.css` file into your application
+
+Then link the CSS file before the closing `</head>` tag of your HTML page or page template.
```
Comment (romaricpascal) [inline source/importing-css-assets-and-javascript/index.html.md.erb:22]: **suggestion** There's less of a clear point of insertion for CSS as there would be with JS, where right before `<body>` makes it download and run last as you'd want. For CSS, it's more the opposite and you want it to load ASAP so it doesn't block the rendering, so as high up in the `<head>` as possible... but [not necessarily all at the top](https://speakerdeck.com/csswizardry/get-your-head-straight?slide=39). So I'd propose something less specific with:

```suggestion
Then link the CSS file inside the `<head>` tag of your HTML page or page template.
```

File: source/importing-css-assets-and-javascript/index.html.md.erb [hunk 7]
```diff
@@ -7,15 +7,36 @@ weight: 70
...
+
+```html
+<head>
+  <!-- // ... -->
+  <link rel="stylesheet" href="<YOUR-STYLESHEETS-FOLDER>/govuk-frontend.min.css">
+</head>
```
Comment (romaricpascal) [inline source/importing-css-assets-and-javascript/index.html.md.erb:28]: If we go less specific with the previous comment, we should be less specific with the example.

```suggestion
<head>
  <!-- // ... -->
  <link rel="stylesheet" href="<YOUR-STYLESHEETS-FOLDER>/govuk-frontend.min.css">
  <!-- // ... -->
</head>
```

**Review by romaricpascal — COMMENTED** [review]
Neat! And nice way to have the same structure for CSS and JS now we ship pre-compiled files. Only a little remark on where we're suggesting to add the CSS in the `<head>`.

File: source/importing-css-assets-and-javascript/index.html.md.erb (hunk 6 above)
Comment (colinrotherham) [reply source/importing-css-assets-and-javascript/index.html.md.erb]: Cheers, nice way to remove any edge cases. Done ✅

**Review by romaricpascal — APPROVED** [review]
Cheers for the update! ⛵

**Pull request merged**